specification/storage/resource-manager/readme.md

```diff
@@ -26,9 +26,39 @@ These are the global settings for the Storage API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2023-04
+tag: package-2023-05
 ```
 
+### Tag: package-2023-05
+
+These settings apply only when `--tag=package-2023-05` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-05'
+input-file:
+  - Microsoft.Storage/stable/2023-05-01/blob.json
+  - Microsoft.Storage/stable/2023-05-01/common.json
+  - Microsoft.Storage/stable/2023-05-01/file.json
+  - Microsoft.Storage/stable/2023-05-01/privatelinks.json
+  - Microsoft.Storage/stable/2023-05-01/queue.json
+  - Microsoft.Storage/stable/2023-05-01/storage.json
+  - Microsoft.Storage/stable/2023-05-01/table.json
+  - Microsoft.Storage/stable/2023-05-01/networkSecurityPerimeter.json
+  - Microsoft.Storage/stable/2023-05-01/storageTaskAssignments.json
+
+directive:
+  - suppress: R3018
+    reason: Existing boolean properties
```

Comment:
>Existing [](http://example.com/codeflow?start=12&length=8)

these are properties in the newly added stoageTaskAssignments json file , how can they be existing boolean properties ?

Reply:
Removed unnecessary suppressions